lib/util/fix-owner.js

```diff
@@ -77,6 +77,11 @@ function fixOwnerSync (cache, filepath) {
   }
   const { uid, gid } = inferOwner.sync(cache)
   getSelf()
+  if (self.uid !== 0) {
+    // almost certainly can't chown anyway
+    return
+  }
+
```

Comment:
caught this while writing the tests, this logic was added for the async version on 2d80af972f0d9b67c19251a20c359f61f868dfb0 but not on the **sync** version